src/move_embargo_args.py

```diff
@@ -137,5 +149,13 @@ def parse_args():
         register_dataset_types=True,
         transfer_dimensions=True,
     )
+    if namespace.log == "True":
+        ids_moved = [
+            dt.dataId.full["exposure"]
+            for dt in scratch_registry.queryDatasets(
```

Comment:
This is going to be a lot more than you expect when it's the production destination.  I think you need to look at `out` to see what was moved.  And then, below, prune only the datasets that were successfully moved in `out`, not all the ones requested.

Reply:
Deleted the log of things we want to move, also now pruning only things that were moved to prompt if move == "True"